src/onboarding/Deployment/DeploymentStepsPanel.js

```diff
@@ -65,10 +68,19 @@ function DeploymentStepsPanel({ transactionsStatus, pending, allSuccess }) {
       </div>
 
       {!allSuccess && (
-        <Info>
+        <Info mode="warning">
           It might take some time before these transactions get processed.
-          Please be patient and{' '}
-          <strong>do not close this window until it finishes.</strong>
+          Please be patient and do not close this window until it finishes.
```

Comment:
I think it's still important to emphasize the "do not close this window" part; while we can offer recoverability, it shouldn't be an accepted flow.